internal/lpastore/client_test.go

```diff
@@ -241,9 +279,11 @@ func TestClientServiceContract(t *testing.T) {
 				"X-Jwt-Authorization": dsl.Regex("Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJvcGcucG9hcy5tYWtlcmVnaXN0ZXIiLCJzdWIiOiJ0b2RvIiwiaWF0Ijo5NDY3NzEyMDB9.teh381oIhucqUD3EhBTaaBTLFI1O2FOWGe-44Ftk0LY", "Bearer .+"),
 			},
 			Body: dsl.Like(map[string]any{
```

Comment:
Should this include `CertificateProvider`?

Reply:
And is it worth including `PeopleToNotify` or are we going for minimum required data?

Reply:
good points